autoreduce_frontend/reduction_viewer/tables.py

```diff
@@ -0,0 +1,96 @@
+import django_tables2 as tables
+from django_tables2 import Table
+from autoreduce_db.reduction_viewer.models import ReductionRun, Experiment
+from autoreduce_frontend.autoreduce_webapp.templatetags.colour_table_row import colour_table_row
+
+
+class ReductionRunTable(Table):
+    def data_status(**kwargs):
+        status = kwargs.get("value", None)
+        if status is None:
+            return "header"
+        else:
+            return "text-" + colour_table_row(status.__str__()) + " run-status"
+
+    run_number = tables.TemplateColumn(
+        '{% load get_run_navigation_queries %} <a href="{% url \'runs:summary\' record.instrument record.run_number record.run_version %}?{% get_run_navigation_queries record.run_number current_page last_instrument_run first_instrument_run %}&page={{current_page}}&per_page={{per_page}}&sort={{ sort }}&filter={{ filtering }}">{{record.run_number}}</a>',
+        attrs={"td": {
+            "class": "run-num-links"
+        }},
+        accessor="run_numbers__run_number")
+
+    status = tables.Column(attrs={"td": {"class": data_status}})
+
+    created = tables.DateTimeColumn(attrs={"td": {"class": "created-dates"}})
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    class Meta:
+        model = ReductionRun
+        row_attrs = {"class": "run-row"}
+        fields = (
+            'run_number',
+            'instrument',
+            'status',
+            'created',
+        )
+        sequence = (
+            'run_number',
+            'instrument',
+            'status',
+            'created',
+        )
+
+
+class ReductionRunSearchTable(Table):
+    def data_status(**kwargs):
+        status = kwargs.get("value", None)
+        if status is None:
+            return "header"
+        else:
+            return "text-" + colour_table_row(status.__str__()) + " run-status"
+
+    run_number = tables.TemplateColumn(
+        '{% load get_run_navigation_queries %} <a href="{% url \'runs:summary\' record.instrument record.run_number record.run_version %}?page={{current_page}}&per_page={{per_page}}&sort={{ sort }}&filter={{ filtering }}">{{record.run_number}}</a>',
```

Comment:
```suggestion
        '{% load get_run_navigation_queries %} <a href="{% url \'runs:summary\' record.instrument record.run_number record.run_version %}?page={{current_page}}&per_page={{per_page}}&sort={{ sort }}&filter={{ filtering }}">{{record.title}}</a>',
```

This shouldn't use `run_number` as that will throw an exception for batch runs, but also the formatting is wrong as the run version + run description/title are missing. It should use `ReductionRun.title()` instead, so that we funnel all the "naming" through the same place

Reply:
Also the `sort` value passed on here seems to be causing an error if I do:

1. Search for a run
2. Open it
3. Click back to instrument

I think it's because then the URL contains `sort=-run_number` which is causing an error in `runs_list.py`.

Reply:
Next PR will be using sort=-run_number to allow the column headings of ReductionRunTable to be sortable. Will look at how bad value is handled in that PR.